books/free-programming-books-langs.md

```diff
@@ -1328,7 +1328,8 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 ### Kotlin
 
-* [Kotlin Notes for Professionals](https://goalkicker.com/KotlinBook/) - Compiled from StackOverflow documentation (PDF)
+* [Kotlin Head First](https://www.pdfdrive.com/download.pdf?id=196919920&h=774e6318fba6d7ab62fe86c91ee433af&u=cache&ext=pdf) (PDF)  
```

Comment:
Remove non-acceptable content. It has copyright and pdfdrive is not the author site

```suggestion
```